ionic_user_interface/src/views/UploadRoute.vue

```diff
@@ -253,18 +253,23 @@ export default defineComponent({
                 toast.present();
               });
 
-            router.push('/home');
+            router.push('/userRoutes/' + getUsername().value);
```

Comment:
You should probably use the name + param format:


![image](https://user-images.githubusercontent.com/46164638/125976839-55a91153-e5b8-4867-9adc-0d79738f5a49.png)